### DISCUSSION TITLE
Still loosing the data

### DISCUSSION THREAD

**Discussion by derfenix** [pgmodeler/pgmodeler#1720] [General]
* Spend a pair of hours to design the database
* Periodically save
* At the last save modeler "Oops"ed and closed
* Project restoring says it can't find sequence for id column and goodbye
* Project file is absent (could you like to save into separate file and replace original on success, huh?)
* Two hours are thrown into the wind

Got the same issue a years ago, wished the app have a more stability, but...

**Comment by rkhaotix** [comment]
Well, I do my best to make pgModeler as stable as possible. Unfortunately, there are bugs that I can´t catch during development and I need help from users to discover what and where the problem is.

Do you have some useful information from a technical point of view so It can help me to track the problem? Version used? OS? Anything that you may find relevant.

**Reply by derfenix** [reply]
Archlinux, pgmodeler v1.0.0_beta
And there is nothing more to attach because of no project file is kept, and no logs from the pgmodeler itself.
I wish you'll add auto backuping before every save, and make restoration more aggressive - it is better to loose part of the project's consistency but restore the project, than leave a user with a blank page on any minor error.

**Reply by rkhaotix** [reply]
Thanks. About the auto backup, pgModeler already does that. Anyway, I'll check the routine and see if  I can find some bugs in it.
